isolate.go

```diff
@@ -103,6 +103,13 @@ func (i *Isolate) Dispose() {
 	i.ptr = nil
 }
 
+// ThrowException will schedule for an exception to be thrown.
```

Comment:
Could you update the Changelog to reflect this new function?

Reply:
Additionally, I wonder how much of the v8 comment for this function should be captured here?
```
   * Schedules an exception to be thrown when returning to JavaScript.  When an
   * exception has been scheduled it is illegal to invoke any JavaScript
   * operation; the caller must return immediately and only after the exception
   * has been handled does it become legal to invoke JavaScript operations.
```

Reply:
Yup! That makes sense. I've pushed a commit with all these fixes.